resources/DK.ke0

```diff
@@ -0,0 +1,142 @@
+################################################################################
```

Comment:
This file is not meant to be part of the distribution, right?

Reply:
(same as below)